doc/newsfragments/qt-version-selection.txt

```diff
@@ -0,0 +1 @@
+If your system has both Qt5 and Qt6 installed you can select the version that will be used by the build by setting the cmake Variable QT_DEFAULT_MAJOR_VERSION to either 5 or 6. If its not set Qt5 will be used.
```

Comment:
I think the wording here could be improved.

It comes across as a bit confusing, and variables should be in backticks. Additionally, the file should not be `.txt`.

Finally, I think grammar could be adjusted; but that might be me nitpicking. I can change this later.

Reply:
Do you have some better wording ? what should the extension be ?

Reply:
I have added to the Qt6.bugfix file this line 
`To Force use of Qt6 set the cmake variable QT_DEFAULT_MAJOR_VERSION to 6.`
I think this is the most sane place to inform builders.

Reply:
I've edited it. This isn't a bug fix, so it's not a good place. It's a feature.